### PR TITLE
{phys}[intel/2018a] BerkeleyGW v1.2.0

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2691.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e055b40cdf1199944b3ffb11a6cdb69f for a full test report.

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.2.0-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']  # requires registration
+sources = ['BGW-%(version)s.tar.gz']
+checksums = [
+    '1305dc8587af666fe437bc2561a9106b2b0bcdbe91980b8f1ae7bbd491ce1e25',  # BGW-1.2.0.tar.gz
+    '7bd3cc229693a1bd5fe6547f55046bee68c8c50f29f8ca6945f443b04ce2ca38',  # BerkeleyGW-1.2.0_fix_intent.patch
+]
+
+patches = ['BerkeleyGW-1.2.0_fix_intent.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.2.0-intel-2018a.eb]: @migueldiascosta Use `%(version)s` here, and move this line above `checksums` (you didn't use `--inject-checksums`?)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2069.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6702f526e37a34c86883ea7656bf1f6d for a full test report.

File: easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.2.0-intel-2018a.eb (hunk 1 above)
Comment (migueldiascosta) [reply easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.2.0-intel-2018a.eb]: I did use `--inject-checksums` when I added the patch, but the source checksum was already there from earlier

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**